### PR TITLE
rolls

### PR DESCRIPTION
**Pull request by siggines**

### CODE REVIEW

File: src/sausage-rolls.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Pork meat / sausage meat
+
+## Directions
+
+1. Place baking paper onto a tray
+2. Whisk eggs
+3. Lay out pastry, brush or pour egg whisk so it lightly covers the pastry
+4. Prepare sausage meat onto pastry
+5. Fold pastry around the meat, apply extra egg whisk to the 'seal' where the pastry is folded together
+6. Bake until golden brown.
```
Comment (DGKSK8LIFE) [inline src/sausage-rolls.md:19]: Think you should elaborate on the directions here. How long should I whisk eggs for? What should the result of whisking the eggs look like?

File: src/sausage-rolls.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Sausage Rolls
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 30-60 min
+
+## Ingredients
+
+- Eggs
+- Puff pastry
+- Pork meat / sausage meat
```
Comment (hallzy) [inline src/sausage-rolls.md:10]: Would be useful to have some measurements or amounts here. ie, how many eggs, how much meat etc.

**Review by hallzy — CHANGES_REQUESTED** [review]

File: src/sausage-rolls.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Sausage Rolls
+
```
Comment (Knarkzel) [inline src/sausage-rolls.md:2]: Missing image.

File: src/sausage-rolls.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 30-60 min
+
```
Comment (Knarkzel) [inline src/sausage-rolls.md:5]: Missing following:

```
- 🍽️ Servings: 4
```

**Pull request merged**